src/api.js

```diff
@@ -91,6 +91,7 @@ const staticExports = {
   ApiResponseClass: response.ApiResponseClass,
   JsonResponse: response.JsonResponse,
   JsonResponseClass: response.JsonResponseClass,
+  fieldProjection: response.fieldProjection,
```

Comment:
Maybe just `projection`?
Or even `retainFields`, `pickFields`, let me know the direction you want to take the naming

Reply:
I was thinking to do this magically when the fields Param exists (potentially with a flag to disable auto pruning). Let's chat about details tomorrow!